ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/notification/NotificationManagerImpl.java

```diff
@@ -100,8 +100,8 @@ public NotificationManagerImpl(NotificationManagerView view,
 
     @Inject
     @PostConstruct
```

Comment:
And I'm not sure that we should link `NotificationManager`'s implementation with custom components especially via method injection.